tests/testthat/test-utilities-output-selections.R

```diff
@@ -34,7 +34,7 @@ test_that("It can add single output by reference", {
   quantities <- addOutputs(parameter, sim)
   expect_equal(length(quantities), 1)
   expect_equal(length(outputSelections$allOutputs), 1)
-  expect_equal(quantities[[1]], parameter)

```

Comment:
I had to change the test because now the output is "Quantity" and not a "Parameter", and the test would fail.

Reply:
I don't understand this. Changing the internal of a function should not change the output

Reply:
Actually now the output is more consistent, because it always returns a Quantity (or a set of quantities). Before, it would return a quantity if the paths are passed, but could return a Parameter or a Molecule in case one of those are passed.

Reply:
I see.. hum